### PR TITLE
Added styling to experimental sdg page

### PR DESCRIPTION
**Pull request by dwnoble**
Screenshot:

Screenshot 2023-05-26 at 3 00 04 PM

### CODE REVIEW

File: experimental/index.html [hunk 1]
```diff
@@ -100,21 +108,269 @@
...
+                  id="Capa_1"
+                  x="0px"
+                  y="0px"
+                  viewBox="0 0 56.966 56.966"
+                  style="enable-background: new 0 0 56.966 56.966"
+                  xml:space="preserve"
+                  width="512px"
+                  height="512px"
+                >
+                  <path
```
Comment (shifucun) [inline experimental/index.html:169]: wonder for button and other UI, can use bootstrap, jquery or other popular components? Then the sdg name could be wrapped in the javascript as an array.

File: experimental/index.html [hunk 2]
```diff
@@ -100,21 +108,269 @@
...
+      <h1 class="font-extralight text-7xl">United States</h1>
+    </div>
+
+    <!-- Development Goals -->
+    <div class="flex relative dc-content mx-auto text-slate-700">
+      <aside
+        class="h-screen sticky top-0 shrink-0 w-0 hidden md:block md:w-80 pt-4 pr-4"
+      >
+        <div
+          class="flex border-l-2 border-transparent hover:!border-slate-300 hover:text-blue-600 px-2 cursor-pointer mb-3"
```
Comment (shifucun) [inline experimental/index.html:193]: Since this is a long list, maybe use a javascript to create each one on the fly? https://jsfiddle.net/ncn7koqx/1/

Comment (dwnoble) [reply experimental/index.html]: Good call- I extracted these to templates

File: experimental/index.html (hunk 1 above)
Comment (dwnoble) [reply experimental/index.html]: that sounds reasonable. Added jquery, lodash, and bootstrap, and removed tailwind

**Comment by dwnoble** [comment]
Updated screenshot using bootstrap:

Screenshot 2023-05-30 at 3 08 17 PM

**Review by shifucun — APPROVED** [review]
Thanks for the update!

**Pull request merged**